### ISSUE TITLE
Missing automated checks in the plugin installation process?

### ISSUE THREAD

**Issue by lonalore** [e107inc/e107#1196]
Maybe I found a bug in plugin-installation process. :(

In the Issue #879, I got this answer:
"... There are many automated checks in the plugin installation process. ..."

Which file contains these kind of checks? Because I defined some (plugin) dependencies in my plugin.xml, see: https://github.com/lonalore/nodejs_pm/blob/master/plugin.xml

... but e107 allows me to install the plugin even if required plugins are missing or not installed.

I think e107 should throw an error if something dependency is missing or not installed, or simply disable the checkbox/button in the current row.

**Comment by CaMer0n** [comment]
Yes, different kind of checks. This part would appear to be incomplete and require some additional code.

**Comment by Moc** [comment]
I consider this a bug as the dependencies should already be checked by the XmlDependencies() method in plugin_class.php. Apparently this is not functioning as intended at the moment.
